### PR TITLE
Fix error messages

### LINKED ISSUE #110: After JDA update, Wylx music playing broken
Wylx will properly select the song and join the VC, but it does not play any music. It stays for about 10-20 seconds then disconnects.

### PR DESCRIPTION
**Pull request by 1Revenger1**
Closes #110

### CODE REVIEW

**Comment by 1Revenger1** [comment]
Doesn't fix music yet

File: src/main/java/com/wylxbot/wylx/Core/Processing/ReactionProcessing.java [hunk 1]
```diff
@@ -45,6 +47,8 @@ public void onMessageReactionRemove(@NotNull MessageReactionRemoveEvent event) {
                 if (e.getErrorResponse() == ErrorResponse.UNKNOWN_ROLE) {
                     removeRole(event, role);
                 }
+            } catch (HierarchyException e) {
+                // TODO: Should probably do smth about this
```
Comment (joshuabrown419) [inline src/main/java/com/wylxbot/wylx/Core/Processing/ReactionProcessing.java:51]: TODO: do the todo

File: src/main/java/com/wylxbot/wylx/Core/Processing/ReactionProcessing.java [hunk 2]
```diff
@@ -56,7 +60,15 @@ private void removeRole(@NotNull GenericMessageReactionEvent event, Role roleToR
     }
 
     private Role checkReactionForMenu(@NotNull GenericMessageReactionEvent event) {
-        Message msg = event.getChannel().retrieveMessageById(event.getMessageId()).complete();
+        Message msg;
+
+        try {
+            msg = event.getChannel().retrieveMessageById(event.getMessageId()).complete();
+        } catch (InsufficientPermissionException e) {
+            // TODO: Should probably do smth about this
```
Comment (joshuabrown419) [inline src/main/java/com/wylxbot/wylx/Core/Processing/ReactionProcessing.java:68]: TODO: do the todo

**Review by joshuabrown419 — CHANGES_REQUESTED** [review]

**Review by joshuabrown419 — COMMENTED** [review]
LGTM

**Review by joshuabrown419 — CHANGES_REQUESTED** [review]
LGTM

**Review by joshuabrown419 — APPROVED** [review]

**Pull request merged**